webapp/src/ts/modules/privacy-policy/privacy-policy.component.ts

```diff
@@ -48,7 +50,7 @@ export class PrivacyPolicyComponent implements OnInit {
     this.accepting = true;
     return this.privacyPoliciesService
       .accept(this.privacyPolicy)
-      .then(() => this.globalActions.setPrivacyPolicyAccepted(true));
+      .then(() => this.globalActions.setPrivacyPolicyAccepted(true))
+      .then(() => this.startupModalsService.showStartupModals());
```

Comment:
I think it was a little more elegant and clean before, when startup-modals was a component and we controlled when it was loaded from the main template and everything was handled in a single place. 

I think the privacy policy component shouldn't need to know about startup modals.

Reply:
On second though, maybe it's not worth the extra code + rendering time to have this additional component and the direct interaction makes things more obvious.

Reply:
coin flip? :D

Reply:
Link to the issue: https://github.com/medic/angular10-migration/issues/38